package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "instant",
+  "version": "0.0.12",
+  "main": "instant.ts",
+  "repository": "git@github.com:openhie/package-starter-kit.git",
+  "author": "Jembi and IntraHealth",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@types/command-line-args": "^5.0.0",
+    "@types/glob": "^7.1.3",
+    "@types/node": "^14.0.22",
+    "command-line-args": "^5.1.1",
+    "env-cmd": "^10.1.0",
+    "glob": "^7.1.6",
+    "ts-node": "^8.10.2",
+    "typescript": "^3.9.6"
```

Comment:
These dependencies seem like something we'd want to lock to a specific version, ideally I'd recommend locking all of them

Reply:
I think this should be part of a team agreement. Currently they're pinned to the major version, which in most cases should be fine (if the packages use semantic versioning properly).
If we're going to pin them at a specific versions, then we need to create a process for updating them and make sure that happens. 
Fine to have that as a practice though.